packages/vscode-extension/package.json

```diff
@@ -736,7 +736,81 @@
         ],
         "properties": {
           "command": {
-            "type": "string"
+            "oneOf": [
+              {
+                "const": "debug-check-prerequisites",
+                "description": "Validate and install prerequisites.\n See https://aka.ms/teamsfx-debug-tasks#check-prerequisites for how to customize following args."
```

Comment:
Any localization support?

Reply:
Use the string in `package.nls.json` instead.